auctions/models.py

```diff
@@ -356,18 +347,17 @@ def notify_matching_offerers(sender, instance, created, **kwargs):
     )
 
     for bid in others_bids:
+        email_context = ({
+            'user': instance.user,
+            'url': instance.issue.url,
+            'offer': bid.offer,
+            'site': current_site,
+            'claim_link': instance.get_absolute_url(),
+        })
+        message = email_template.render(email_context)
         send_mail(
-            "[codesy] %(user)s has claimed payout for %(url)s" %
-            (
-                {'user': instance.user, 'url': instance.issue.url}
-            ),
-            OFFERER_NOTIFICATION_EMAIL_STRING.format(
-                user=instance.user,
-                url=instance.issue.url,
-                offer=bid.offer,
-                site=current_site,
-                claim_link=instance.get_absolute_url()
-            ),
+            "[codesy] A claim needs your vote!",
```

Comment:
A nit-pick for a follow-up issue ...

Change claim email subject to "We need your vote by %(claim_end)s" to give recipients a deadline/time-scale before their vote will be auto-approved.